modules/criteoBidAdapter.js

```diff
@@ -224,9 +224,9 @@ function publisherTagAvailable() {
 function buildContext(bidRequests, bidderRequest) {
   let referrer = '';
   if (bidderRequest && bidderRequest.refererInfo) {
-    referrer = bidderRequest.refererInfo.referer;
+    referrer = bidderRequest.refererInfo.page;
   }
-  const queryString = parseUrl(referrer).search;
+  const queryString = parseUrl(bidderRequest?.refererInfo?.topmostLocation).search;
```

Comment:
I think it should be referrerInfo.page here too
What made you think it was better to use topmostlocation ?

Reply:
Nop indeed it makes sense, let's keep topmostlocation.

Thanks !